### PR TITLE
[copy update] WD-3785 copy update on `/landscape/pricing`

### PR DESCRIPTION
**Pull request by pedoch**
## Done

- Copy update for `/landscape/pricing`

## QA

- Check out the [demo here](https://ubuntu-com-12878.demos.haus/landscape/pricing) and compare it with the [copy doc](https://docs.google.com/document/d/1pESmRniuiaDHkbzY_60GlOJdCiH2pZ9erI9KV49eDyM/edit#).

## Issue / Card

- [WD-3785](https://warthogs.atlassian.net/browse/WD-3785)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-3785]: https://warthogs.atlassian.net/browse/WD-3785?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by petesfrench** [comment]
@pedoch I have left a comment in the doc for Rajan, it may result in some changes to the page

File: templates/landscape/pricing.html [hunk 1]
```diff
@@ -105,16 +105,16 @@ <h1>Choose the best Landscape for you</h1>
...
           <td>Free for up to 10 machines for personal use, or evaluation purposes.<br /><br />Beyond the free tier, <a href="/pro">Ubuntu Pro</a> is required for more than 10 machines on self-hosted Landscape</td>
           <td>Canonical's standard <a href="/managed/apps">Managed Apps</a> pricing applies</td>
         </tr>
         <tfoot>
           <tr>
             <td></td>
-          <td><a href="/pro/subscribe">Pricing calculator for Ubuntu Pro</a></td>
-            <td><a href="/landscape/install">Set up your self-hosted Landscape</a></td>
-            <td><a href="/contact-us/form?product=landscape" class="p-button--positive is-dense js-invoke-modal">Contact us</a> to get started</td>
+            <td><a href="#get-in-touch" class="p-button--positive is-dense js-invoke-modal">Get started</a></td>
```
Comment (petesfrench) [inline templates/landscape/pricing.html:115]:
```suggestion
            <td><a href="/contact-us/form?product=landscape" class="p-button--positive is-dense js-invoke-modal">Get started</a></td>
```
can we keep these as fallbacks in the case that JS is not enabled?

File: templates/landscape/pricing.html [hunk 2]
```diff
@@ -105,16 +105,16 @@ <h1>Choose the best Landscape for you</h1>
...
+            <td><a href="/landscape/install">Install self-hosted Landscape</a></td>
+            <td><a href="#get-in-touch" class="p-button--positive is-dense js-invoke-modal" data-form-location="/shared/forms/interactive/landscape-managed" data-form-id="5054" data-lp-id="" data-return-url="/landscape/managed#success" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">Get started</a></td>
```
Comment (petesfrench) [inline templates/landscape/pricing.html:117]:
```suggestion
            <td><a href="/contact-us/form?product=landscape" class="p-button--positive is-dense js-invoke-modal" data-form-location="/shared/forms/interactive/landscape-managed" data-form-id="5054" data-lp-id="" data-return-url="/landscape/managed#success" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">Get started</a></td>
```

**Review by petesfrench — CHANGES_REQUESTED** [review]
Do we need to include the two forms at the top of the table?

**Comment by pedoch** [comment]
Seems like they were just included, to be part of this update. So I'd say yes.

**Comment by pedoch** [comment]
@petesfrench the task was updated all changes on the copy doc are going with this PR except the change [https://warthogs.atlassian.net/browse/WD-3792](https://warthogs.atlassian.net/browse/WD-3792) highlighted in this task (which is essentially the contact form/buttons on the Landscape SaaS column).

**Review by petesfrench — APPROVED** [review]
LGTM

**Pull request merged**